disorder-fsm/src/Disorder/FSM.hs

```diff
@@ -1,9 +1,14 @@
 {-# LANGUAGE NoImplicitPrelude #-}
 {-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
```

Comment:
The bug re. the combination of generalized newtype deriving and type families was fixed with the addition of roles afaict.